kafka-observer/run.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+set -euo pipefail
+
+BASEDIR="$(dirname "$0")"
+
+# Resources vars
+CM_TOPICJOBS_NAME="topic-to-jobs"
+AUTH_SECRETS_NAME="authentication-secrets"
+
+# Consumers vars
+CONSUMER_JOB_NAME_1="payments-consumer"
+CONSUMER_JOB_NAME_2="shipping-consumer"
+
+# Observer vars
+OBSERVER_JOB_NAME="ce-kafka-observer"
+OBSERVER_JR_NAME="kafka-observer"
```

Comment:
I  suggest to leave up the naming of the jobrun to the job controller and not specify it. My personal opinion biased through year of using Code Engine: It is a bit confusing that the jobrun does not share the job template name as prefix

Reply:
I prefer a fixed name, not autogenerated. I modified the name to match the job one.